Framework/WorkflowAlgorithms/src/AlignAndFocusPowder.cpp

```diff
@@ -161,6 +166,19 @@ void AlignAndFocusPowder::init() {
                   "pulse to remove. 0 disables");
   auto mustBePositive = std::make_shared<BoundedValidator<double>>();
   mustBePositive->setLower(0.0);
+
+  std::vector<std::string> allowedResonanceUnits({"Energy", "Wavelength"});
+  declareProperty(PropertyNames::RESONANCE_UNITS, allowedResonanceUnits.back(),
+                  std::make_shared<StringListValidator>(allowedResonanceUnits),
+                  "Units for resonances to be filtered in. "
+                  "The data will be converted to these units temporarily to filter.");
+  declareProperty(std::make_unique<ArrayProperty<double>>(PropertyNames::RESONANCE_LOWER_LIMITS),
+                  "Minimum values to filter absorption resonance. This must have same number of values as "
+                  "ResonanceFilterUpperLimits. Default behavior is to not filter.");
```

Comment:
Could you be more specific about what is 'default behavior'?  Users of this method may get a little bit confused.